### PR TITLE
OSSM-2006 Fix multiNamespaceInformer.HasSynced()

### PR DESCRIPTION
**Pull request by luksa**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -2,23 +2,21 @@ module maistra.io/api
...
 	github.com/spf13/pflag v1.0.5
 	google.golang.org/grpc v1.42.0
-	k8s.io/api v0.22.2
-	k8s.io/apiextensions-apiserver v0.22.2
-	k8s.io/apimachinery v0.22.2
-	k8s.io/client-go v0.22.2
-	k8s.io/code-generator v0.22.2
-	sigs.k8s.io/controller-runtime v0.9.6
-	sigs.k8s.io/controller-tools v0.6.2
+	k8s.io/api v0.23.5
```
Comment (jwendell) [inline go.mod:13]: Did you change this manually?

**Review by jwendell — COMMENTED** [review]
It looks like we need to setup prow jobs for 2.3

File: go.mod (hunk 1 above)
Comment (luksa) [reply go.mod:13]: I only changed the xns-informer dependency. We forgot to update maistra/api during the rebase, that's the reason for these changes.

Comment (luksa) [reply go.mod:13]: I've submitted another PR for these other changes: https://github.com/maistra/api/pull/33

File: go.mod [hunk 2]
```diff
@@ -2,23 +2,21 @@ module maistra.io/api
...
+	k8s.io/apiextensions-apiserver v0.22.4
```
Comment (jwendell) [inline go.mod:14]: I wonder if it makes sense to align those versions with what we have in istio 2.3: https://github.com/maistra/istio/blob/maistra-2.3/go.mod

(controller-runtime below too)

Comment (luksa) [reply go.mod:14]: Yeah. I pointed out to @jewertow that maistra/api needs to be updated too as part of the rebase and he said he'd do it in the next sprint. That's why, for now, I've only updated the minimum in #33.

Comment (jwendell) [reply go.mod:14]: ok, let's wait for #33 to be merged so you can rebase this pr

**Pull request closed without merging**